test/support/admin_edition_controller_test_helpers.rb

```diff
@@ -293,7 +293,8 @@ def should_allow_editing_of(edition_type)
         put :update, params: {
           id: edition,
           edition: {
-            title: '   my title    '
+            title: '   my title    ',
+            previously_published: false
```

Comment:
Should there be a trailing comma here (like https://github.com/alphagov/whitehall/pull/4076/files#diff-669351f665ea3e569b4809a078a4cf9dR16)?  I thought the linter enforced a common style for this.

Reply:
Yes there should! Thanks, amended now.

Reply:
The linter only goes through the lines you're modifying I think! It'd be very noisy if not as it was introduced a few years into the life of the app :-)

Reply:
On this subject, I've got a PR to fix/ignore the few remaining linting issues, and then change the Jenkins configuration to check the entire codebase for any change: https://github.com/alphagov/whitehall/pull/4113